recipes/nbgrader/post-link.sh

```diff
@@ -0,0 +1,5 @@
+#{
+#  "${PREFIX}/bin/jupyter-nbextension" enable --sys-prefix --py nbgrader
+#  "${PREFIX}/bin/jupyter-serverextension" enable --sys-prefix --py nbgrader
+#} >>"$PREFIX/.messages.txt" 2>&1
+# XXX Placeholder for next nbgrader release
```

Comment:
@ocefpaf thanks for the feedback. These files will be needed to post-link the Jupyter extensions after install and pre-unlink before uninstall. The code is commented out as we have only just implemented this in nbgrader and this will be available in the next feature release.